src/linux.rs

```diff
@@ -73,7 +72,7 @@ pub fn linux_ifaddresses(if_name: &str) -> Result<IfAddrs, Box<dyn std::error::E
                 if let Some(inet_addr) = address.as_sockaddr_in() {
                     add_to_types_mat(
                         AF_INET,
-                        &ip_to_string(inet_addr.ip()),
+                        &inet_addr.ip(),
```

Comment:
This breaks the address display. Please revert

Reply:
the returned value is not `u32` anymore starting nix 0.28.0, it's [`core::net::Ipv4Addr`](https://doc.rust-lang.org/stable/core/net/struct.Ipv4Addr.html)

Reply:
Yep, just realized.